frontends/concrete-python/concrete/fhe/compilation/circuit.py

```diff
@@ -109,43 +82,18 @@ def mlir(self) -> str:
         Returns:
             str: textual representation of the MLIR module
         """
-        return str(self.mlir_module).strip()
+        return str(self._module.mlir_module).strip()
 
     def enable_fhe_simulation(self):
         """
         Enable FHE simulation.
         """
 
-        if not hasattr(self, "simulator"):
```

Comment:
? Nothing todo

Reply:
There is nothing to do since simulation uses lazy loading .

Reply:
so maybe just access to the simulation_runtime.val to force the compilation

Reply:
Ah ok, I did not get your comment. Sure, will do !